documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Access the Administration Console"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 1
+description: "Access the WebLogic Admin Console to manage domains running in Kubernetes."
+---
+
+You can use the WebLogic Server Administration Console to monitor and manage a WebLogic domain running in Kubernetes.
+
+{{% notice note %}} Do not use the WebLogic Server Administration Console to start or stop servers, or for scaling clusters. See [Starting and stopping servers]({{< relref "/managing-domains/domain-lifecycle/startup#starting-and-stopping-servers" >}}) and [Scaling]({{< relref "/managing-domains/domain-lifecycle/scaling.md" >}}). In addition, if your domain home type is either [Domain in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}) or [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}), then do not use the Administration Console to make changes to the WebLogic domain configuration as these changes are ephemeral and will be lost when servers restart. See [Choose a domain home source type]({{< relref "/managing-domains/choosing-a-model/_index.md" >}}).
+{{% /notice %}}
```

Comment:
I know this pull is about the admin console, but I cannot find this same note in the remote-console doc. I assume it should be there too.

Reply:
I'm not very familiar with remote-console and not sure if users can start or stop servers and scale clusters with remote console.